db/schema.rb

```diff
@@ -10,12 +10,12 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_06_12_094303) do
+ActiveRecord::Schema[7.0].define(version: 2023_06_13_123131) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "messages", force: :cascade do |t|
-    t.string "greeting"
+  create_table "greetings", force: :cascade do |t|
+    t.string "message"
```

Comment:
- [ ] While fetching the random message API endpoint the error in the image below is raised:

![image](https://github.com/Thoth1111/Hello-Rails-React/assets/68796232/f6fea378-e168-43ea-a3cd-506329188032)

This problem happens because you forgot to add greetings seeds to your database, As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=your%20messages%20and-,create%205%20different%20greetings,-.), please add 5 different greetings messages.